### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hsiaoyi0504**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +27,8 @@ Download to plugins/ColorByType and change the type to `ColorByType/View/Track/C
       },
 ```
 
+You can also do this when using **flatfile-to-json.pl** with **--trackType ColorByType/View/Track/ColorByTypeDraggable** flag.
```
Comment (dytk2134) [inline README.md]: Need to add more description about which situation can use this flag.

Comment (hsiaoyi0504) [reply README.md]: What kind of description ? You can supplement it, though. I wrote this totally based on the description you told me.

**Comment by dytk2134** [comment]
I will add more detail about when can user use **flatfile-to-json.pl** with **--trackType ColorByType/View/Track/ColorByTypeDraggable** flag.

**Comment by hsiaoyi0504** [comment]
Ok, thanks, but I think we can still merge this.

**Comment by hsiaoyi0504** [comment]
@dytk2134 will update README.md together in #3. I have edited the commit. Please review again and merge if you think it's ok.

**Comment by dytk2134** [comment]
Will merge to https://github.com/NAL-i5K/ColorByType/pull/3, see https://github.com/NAL-i5K/ColorByType/pull/4

**Comment by hsiaoyi0504** [comment]
Thanks.

**Pull request closed without merging**